test/lib/mock_tcollector.js

```diff
@@ -0,0 +1,112 @@
+import fs from 'fs';
+import path from 'path';
+import TChannelThrift from 'tchannel/as/thrift';
```

Comment:
why do we need mock tcollector? The whole point of doing opentracing over TChannel is to avoid using tchannel's own span reporting facilities.